atmo/settings.py

```diff
@@ -114,6 +120,21 @@
     )
 }
 
+REDIS_URL = config(
+    'REDIS_URL',
+    cast=parse.urlparse
+)
+
+RQ_QUEUES = {
+    'default': {
+        'HOST': REDIS_URL.hostname,
+        'PORT': REDIS_URL.port,
+        'DB': 0,
```

Comment:
Just in theory, in case we ever require to specify a specific db, we could use the path component of the `REDIS_URL` value. I think it's okay to stick to 0 for now